package.json

```
@@ -8,9 +8,9 @@
 		"build": "webpack --node-env production --progress",
 		"dev": "webpack --node-env development --progress",
 		"watch": "webpack --node-env development --progress --watch",
-		"test:unit": "TZ=UTC jest",
+		"test": "TZ=UTC jest",
+		"test:watch": "TZ=UTC jest --watchAll",
```

```suggestion
		"test:watch": "TZ=UTC jest --watch",
```
Maybe simple --watch is enough? If you change a common file, like `constants.js`, it triggers 19 / 33 tests to run, only for files that only use constants

---

sounds good!